lib/ssl/src/ssl_cipher.erl

```diff
@@ -214,16 +214,16 @@ build_cipher_block(BlockSz, Mac, Fragment) ->
     [Fragment, Mac, padding_with_len(TotSz, BlockSz)].
 
 block_cipher(Fun, BlockSz, #cipher_state{key=Key, iv=IV} = CS0,
-	     Mac, Fragment, {3, N})
-  when N == 0; N == 1 ->
+	     Mac, Fragment, Version)
+  when Version == ?'SSL-3.0'; Version == ?'TLS-1.0' ->
```

Comment:
We have dropped support for SSL-3.0, we only keep it around to allow legacy hello messages that will negotiate higher versions so it should no longer be needed here.